gravity-forms/gw-inventory.php

```diff
@@ -521,5 +525,5 @@ class GWLimitBySum extends GW_Inventory { }
 	'not_enough_stock_message' => 'You ordered %1$s tickets. There are only %2$s tickets left.',
 	'approved_payments_only'   => false,
 	'hide_form'                => false,
-	'enable_notifications'     => true,
+	'enable_notifications'     => false,
```

Comment:
Curious, what's the reason for this change?

Reply:
Using this option requires the GW_Notificaiton_Event class or it generates a doing it wrong notice. It should be disabled by default.